src/scheduled.service.js

```diff
@@ -13,14 +13,14 @@ plugin.service('wgnScheduledWebhook', ['wgnWebhookCommon', function (webhookComm
 		var defaults = {
 			frequency: 'daily',
 			description: 'Scheduled webhook for wgn',
-			start: new Date(),
-			timezone: 'America/New_York',
+			start: moment().format('YYYY-MM-DD[T]HH:mm:ss'),
```

Comment:
This is passing the time in the client browser's timezone, but telling the API to treat it as UTC. The API default is UTC+now, so these default's aren't really necessary. I did some testing on the API and this will cause the scheduling to be off.

Reply:
@tehpsalmist I'll defer to @wdmny on this one and I agree 100% that `new Date()` is stupid and needs to be fixed too. :)

Reply:
@wdmny @alexweber As you can see, I've removed the timezone and start time from the defaults but there is still wacky behavior afoot in this module. The only solution I've found so far is to pass either `{ timezone: null }` or `{timezone: 'UTC'}` via the frontend-config webhook method. For whatever reason, this prevents the start time from being off by 4 hours. But simply passing nothing at all for start && timezone still results in an errant start time. When I pass nothing for these props in the backend, the start time comes out perfectly. This only happens in the frontend, and I'm running out of ideas for why that would be.

Reply:
@wdmny spare some change?